pkg/gadgets/trace/open/tracer/tracer_test.go

```diff
@@ -115,6 +115,11 @@ func TestOpenTracer(t *testing.T) {
 		validateEvent   func(*testing.T, *utilstest.RunnerInfo, int, []types.Event)
 	}
 
+	normalize_cgroupid := func(e *types.Event) {
+		e.CgroupID = 0
```

Comment:
Out of curiosity, is adding the CgroupID to the `RunnerInfo` struct that complex?

cc @mauriciovasquezbernal @alban

Reply:
As opposed to cgroup-v1, cgroup-v2 are normally per process and not per thread (except with the "thread mode", see `man 7 cgroups`, but that's too complex for here). Since our testing framework is just using one thread for the workload, testing the cgroup-v2 id here would not work.